test/rtos_drivers/hil_add/src/individual_tests/uart/local/tx_write_rx_read_test.c

```diff
@@ -1,4 +1,5 @@
-// Copyright 22022 XMOS LIMITED.
+// Copyright 2023 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
```

Comment:
Duplicate line.

Reply:
Fixed in https://github.com/xmos/fwk_rtos/pull/219.